### ISSUE TITLE
Can't stop a shutter switch at the desired position

### ISSUE THREAD

**Issue by m3tc0n** [rospogrigio/localtuya#1519]
Hello,

I have a problem with my shutter switch in LocalTuya.

I recently found out that when I press the 'stop button' the shutter stops, but immediately opens again. So I cannot use the 'stop button' to stop the shutter at the desired position.

When I set the position of the shutter to the desired position via the slider, the problem does not occur.

I have set the switch via the following conditions in LocalTuya:

Type = cover
ID = 1
Open_Close_Stop Commands Set = open_close_stop
Positioning mode = position
Current position = 2(value:100)
Set position = 2(value:100)

All this used to work properly, but for some time now it hasn't.

In short, how can I get the 'stop button' to stop the shutter at the desired position?

Thanks in advance for your help.
